tmt/checks/dmesg.py

```diff
@@ -13,6 +14,13 @@
     from tmt.steps.execute import TestInvocation
 
 TEST_POST_DMESG_FILENAME = 'dmesg-{event}.txt'
+FAILURE_PATTERNS = [
```

Comment:
An idea from me, could we have also an environment variable to extend the list of failure patterns for easier extension. If we would need to extend the list now, a new tmt version needs to happen, with the environment variable for extending the patterns the users would have a workaround.

Reply:
ok, I can come back later